requirements.txt

```diff
@@ -5,3 +5,7 @@ elasticsearch==7.16.3
 PyYAML==6.0
 aws_lambda_typing==2.16.3
 ujson==5.6.0
+requests==2.25.1
+responses>=0.22.0
```

Comment:
I guess that `responses`, `testcontainers` and `amazon-kclpy` should go to `requirements-test.txt`

also https://github.com/elastic/elastic-serverless-forwarder/pull/210/files#diff-dd6813d8bd799e4b5e2af9f37e87fadc28004eecb26dc652ade372ce3e12d153R11 imports `urllib3`, we should add to `requirements.txt`